app/content/models/news.py

```diff
@@ -17,6 +19,8 @@ class News(BaseModel, OptionalImage, BasePermissionModel):
         related_name="created_news",
     )
     body = models.TextField()
+    emojis_allowed = models.BooleanField(default=False)
+    reactions = GenericRelation(Reaction)
```

Comment:
Er det en grunn til at dere har benyttet GenericRelation vs.ManyToManyField?

Reply:
For å gjøre modellen generisk slik at den enkelt kan kobles til andre modeller som event og comment senere, funker dette bedre.

Vi gjør det samme med Kommentarfelt